tests/static_lib/withapp/premake5.lua

```diff
@@ -29,7 +29,7 @@ project "ninjatestprj_app"
 		defines {"NDEBUG"}
 		optimize "On"
 
-project "ninjatestprj_lib test1"
+project "ninjatestprj_lib_test1"
```

Comment:
Might be seen as typo, but handling of space in project in now supported by this module.